imguralbum.py

```diff
@@ -158,7 +159,7 @@ def save_images(self, foldername=False):
                 try:
                     urllib.request.urlretrieve(image_url, path)
                 except:
-                    print ("Download failed.")
+                    print ("Download failed.\n"+format_exc())
```

Comment:
Shouldn't it be `traceback.print_exc()`? (https://docs.python.org/3/library/traceback.html)